parsl/tests/test_providers/test_kubernetes_provider.py

```diff
@@ -0,0 +1,101 @@
+import re
+from unittest import mock
+
+import pytest
+
+from parsl.providers.kubernetes.kube import KubernetesProvider
+
+try:
+    import kubernetes
+except ImportError:
+    KUBERNETES_ENABLED = False
+else:
+    KUBERNETES_ENABLED = True
+
+require_kubernetes = pytest.mark.skipif(
+    not KUBERNETES_ENABLED, reason="kubernetes module not available"
+)
```

Comment:
this isn't working right by the looks of failing CI.

but I am massively in favour of the tests declaring "we expect this to not work" using `pytest.mark.whatever`  rather than autodetecting that the test will fail and not running the test... (for example, if this autodetection was working properly, CI would be passing without running newly introduced tests, I think?  in which case, you might as well not include them in this PR)

Reply:
Good catch; I forgot to update the `config_local_test` target of Makefile. I also removed the import check.